certbot/certbot/_internal/cert_manager.py

```diff
@@ -305,7 +315,8 @@ def get_certnames(config, verb, allow_multiple=False, custom_prompt=None):
             raise errors.Error("No existing certificates found.")
         if allow_multiple:
             if not custom_prompt:
-                prompt = "Which certificate(s) would you like to {0}?".format(verb)
+                prompt = ("Which certificate or certificates would you like to "
```

Comment:
I don't see a reason to expand this phrasing, personally.  If it is a dislike of `(s)`, I think prompting "Which certificates would you like to delete?" make sense, even if there is only a single certificate available or chosen.